files/settings-default.cfg

```diff
@@ -1,244 +1,709 @@
-# WARNING: Editing this file might have no effect, as these
-# settings are overwritten by your user settings file.
+# WARNING: Editing this file might have no effect, as these settings
+# are overwritten by your user settings file.  Your user settings file
+# varies with your operating system:
+#
+# Linux: $HOME/.config/openmw
+# Mac: $HOME/Library/Preferences/openmw
+# Windows: C:\Users\Username\Documents\my games\openmw
+#   This path may vary depending on your installation hard drive, your
+#   Windows username, and your default language.
+#
+# Additionally, the user settings file is often written to disk when
+# exiting OpenMW, so comments and changes to that file may also be
+# discarded after running OpenMW.  While most changes to the file will
+# reflect setting changes made in game, some settings can have a wider
+# range of values in the settings file than the GUI settings widgets
+# allow.  You may want to exercise some caution and backup this file
+# when editing it by hand.
```

Comment:
Agree with the caution part. Maybe we can add another sentence: "Certain values may cause the game to malfunction or crash".